### ISSUE TITLE
[Bug]: dbt_artifacts own code giving syntax error

### ISSUE THREAD

**Issue by calkan-cue** [brooklyn-data/dbt_artifacts#331]
## Overview

The package's own code throwing error. 
## How to reproduce

dbt build --select dbt_artifacts

## Expected behaviour
I was expecting it to run fine and create necessary tables but instead it throws syntax errors. 

## Screenshots

> 07:00:48  Completed with 11 errors and 0 warnings:
> 07:00:48  
> 07:00:48  Database Error in model stg_dbt__model_executions (models/staging/stg_dbt__model_executions.sql)
> 07:00:48    Syntax error: Expected ")" but got "(" at [19:148]
> 07:00:48    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__model_executions.sql
> 07:00:48  
> 07:00:48  Database Error in model stg_dbt__exposures (models/staging/stg_dbt__exposures.sql)
> 07:00:48    Syntax error: Expected ")" but got "(" at [19:148]
> 07:00:48    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__exposures.sql
> 07:00:48  
> 07:00:48  Database Error in model stg_dbt__seed_executions (models/staging/stg_dbt__seed_executions.sql)
> 07:00:48    Syntax error: Expected ")" but got "(" at [19:148]
> 07:00:48    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__seed_executions.sql
> 07:00:48  
> 07:00:48  Database Error in model stg_dbt__snapshots (models/staging/stg_dbt__snapshots.sql)
> 07:00:48    Syntax error: Expected ")" but got "(" at [19:148]
> 07:00:48    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__snapshots.sql
> 07:00:48  
> 07:00:48  Database Error in model stg_dbt__models (models/staging/stg_dbt__models.sql)
> 07:00:48    Syntax error: Expected ")" but got "(" at [19:148]
> 07:00:48    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__models.sql
> 07:00:48  
> 07:00:48  Database Error in model stg_dbt__snapshot_executions (models/staging/stg_dbt__snapshot_executions.sql)
> 07:00:48    Syntax error: Expected ")" but got "(" at [19:148]
> 07:00:48    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__snapshot_executions.sql
> 07:00:48  
> 07:00:48  Database Error in model stg_dbt__seeds (models/staging/stg_dbt__seeds.sql)
> 07:00:48    Syntax error: Expected ")" but got "(" at [19:148]
> 07:00:48    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__seeds.sql
> 07:00:48  
> 07:00:48  Database Error in model stg_dbt__test_executions (models/staging/stg_dbt__test_executions.sql)
> 07:00:48    Syntax error: Expected ")" but got "(" at [19:148]
> 07:00:48    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__test_executions.sql
> 07:00:48  
> 07:00:48  Database Error in model stg_dbt__sources (models/staging/stg_dbt__sources.sql)
> 07:00:48    Syntax error: Expected ")" but got "(" at [19:148]
> 07:00:48    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__sources.sql
> 07:00:48  
> 07:00:48  Database Error in model stg_dbt__tests (models/staging/stg_dbt__tests.sql)
> 07:00:48    Syntax error: Expected ")" but got "(" at [19:148]
> 07:00:48    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__tests.sql
> 07:00:48    

## Environment

Results of running `dbt --version`:

```sh
Core:
  - installed: 1.4.5
  - latest:    1.5.0 - Update available!

  Your version of dbt-core is out of date!
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation

Plugins:
  - bigquery: 1.4.1 - Update available!

  At least one plugin is out of date or incompatible with dbt-core.
  You can find instructions for upgrading here:
```

Please paste the contents of your `packages.yml` file here:  

```yml
 packages:
  - package: dbt-labs/codegen
    version: 0.9.0
  - package: dbt-labs/dbt_project_evaluator
    version: 0.4.1
  - package: dbt-labs/dbt_external_tables
    version: 0.8.2
  - package: brooklyn-data/dbt_artifacts
    version: 2.3.0
  - package: fivetran/shopify
    version: [">=0.8.0", "<0.9.0"]
  - package: fivetran/ad_reporting
    version: [">=1.3.0", "<1.4.0"]
  - package: fivetran/netsuite
    version: [">=0.7.0", "<0.8.0"]
  - package: fivetran/salesforce
    version: [">=0.8.0", "<0.9.0"]
  - package: dbt-labs/metrics
    version: [">=1.4.0", "<1.5.0"]

```

**Comment by jared-rimmer** [comment]
Hi @calkan-cue 

Thanks for raising this issue!

I've been looking into this and I'm unable to replicate with the same settings as you.

Are you able to do a `dbt clean && dbt deps` and let me know the outcome please?

Furthermore, if you able to copy the full logs that could help me pinpoint the issue.

**Comment by calkan-cue** [comment]
```sh
dbt clean 
dbt deps
dbt build --select dbt_artifacts
```

```yaml
dbt_project.yml
  surrogate_key_treat_nulls_as_empty_strings: true #turn on legacy behavior for dbt uttil generate_surrogate_key

```

I am wondering if it can be related with surrogate keys null rule issue but  not sure.
After I run again in the way you said the results are same.

**Comment by calkan-cue** [comment]
> (.venv) user@ConsOle dbt % dbt clean
> 13:14:07  Running with dbt=1.4.5
> 13:14:07  Checking target/*
> 13:14:08  Cleaned target/*
> 13:14:08  Checking dbt_packages/*
> 13:14:08  Cleaned dbt_packages/*
> 13:14:08  Finished cleaning all paths.
> (.venv) user@ConsOle dbt % dbt deps 
> 13:14:54  Running with dbt=1.4.5
> 13:15:03  Installing dbt-labs/codegen
> 13:15:04    Installed from version 0.9.0
> 13:15:04    Up to date!
> 13:15:04  Installing dbt-labs/dbt_project_evaluator
> 13:15:05    Installed from version 0.4.1
> 13:15:05    Updated version available: 0.6.0
> 13:15:05  Installing dbt-labs/dbt_external_tables
> 13:15:05    Installed from version 0.8.2
> 13:15:05    Updated version available: 0.8.4
> 13:15:05  Installing brooklyn-data/dbt_artifacts
> 13:15:06    Installed from version 2.3.0
> 13:15:06    Up to date!
> 13:15:06  Installing fivetran/shopify
> 13:15:06    Installed from version 0.8.1
> 13:15:06    Up to date!
> 13:15:06  Installing fivetran/ad_reporting
> 13:15:07    Installed from version 1.3.1
> 13:15:07    Up to date!
> 13:15:07  Installing fivetran/netsuite
> 13:15:08    Installed from version 0.7.1
> 13:15:08    Updated version available: 0.8.0
> 13:15:08  Installing fivetran/salesforce
> 13:15:09    Installed from version 0.8.0
> 13:15:09    Updated version available: 0.9.1
> 13:15:09  Installing dbt-labs/metrics
> 13:15:09    Installed from version 1.4.1
> 13:15:09    Up to date!
> 13:15:09  Installing dbt-labs/dbt_utils
> 13:15:10    Installed from version 1.1.0
> 13:15:10    Up to date!
> 13:15:10  Installing fivetran/shopify_source
> 13:15:11    Installed from version 0.8.2
> 13:15:11    Up to date!
> 13:15:11  Installing fivetran/apple_search_ads
> 13:15:11    Installed from version 0.2.0
> 13:15:11    Up to date!
> 13:15:11  Installing fivetran/snapchat_ads
> 13:15:12    Installed from version 0.5.1
> 13:15:12    Up to date!
> 13:15:12  Installing fivetran/facebook_ads
> 13:15:13    Installed from version 0.6.0
> 13:15:13    Up to date!
> 13:15:13  Installing fivetran/google_ads
> 13:15:14    Installed from version 0.9.1
> 13:15:14    Up to date!
> 13:15:14  Installing fivetran/pinterest
> 13:15:14    Installed from version 0.7.0
> 13:15:14    Up to date!
> 13:15:14  Installing fivetran/linkedin
> 13:15:15    Installed from version 0.7.0
> 13:15:15    Up to date!
> 13:15:15  Installing fivetran/microsoft_ads
> 13:15:16    Installed from version 0.6.0
> 13:15:16    Up to date!
> 13:15:16  Installing fivetran/tiktok_ads
> 13:15:16    Installed from version 0.3.0
> 13:15:16    Up to date!
> 13:15:16  Installing fivetran/twitter_ads
> 13:15:17    Installed from version 0.6.0
> 13:15:17    Up to date!
> 13:15:17  Installing fivetran/amazon_ads
> 13:15:18    Installed from version 0.1.0
> 13:15:18    Up to date!
> 13:15:18  Installing fivetran/reddit_ads
> 13:15:20    Installed from version 0.1.0
> 13:15:20    Up to date!
> 13:15:20  Installing fivetran/netsuite_source
> 13:15:21    Installed from version 0.6.2
> 13:15:21    Up to date!
> 13:15:21  Installing fivetran/salesforce_source
> 13:15:21    Installed from version 0.6.0
> 13:15:21    Updated version available: 0.7.0
> 13:15:21  Installing fivetran/fivetran_utils
> 13:15:22    Installed from version 0.4.6
> 13:15:22    Up to date!
> 13:15:22  Installing dbt-labs/spark_utils
> 13:15:22    Installed from version 0.3.0
> 13:15:22    Up to date!
> 13:15:22  Installing calogica/dbt_expectations
> 13:15:23    Installed from version 0.8.5
> 13:15:23    Up to date!
> 13:15:23  Installing fivetran/apple_search_ads_source
> 13:15:24    Installed from version 0.2.1
> 13:15:24    Up to date!
> 13:15:24  Installing fivetran/snapchat_ads_source
> 13:15:24    Installed from version 0.5.0
> 13:15:24    Up to date!
> 13:15:25  Installing fivetran/facebook_ads_source
> 13:15:26    Installed from version 0.6.0
> 13:15:26    Up to date!
> 13:15:26  Installing fivetran/google_ads_source
> 13:15:26    Installed from version 0.9.2
> 13:15:26    Up to date!
> 13:15:26  Installing fivetran/pinterest_source
> 13:15:27    Installed from version 0.7.1
> 13:15:27    Up to date!
> 13:15:27  Installing fivetran/linkedin_source
> 13:15:28    Installed from version 0.7.0
> 13:15:28    Up to date!
> 13:15:28  Installing fivetran/microsoft_ads_source
> 13:15:28    Installed from version 0.7.0
> 13:15:28    Up to date!
> 13:15:28  Installing fivetran/tiktok_ads_source
> 13:15:29    Installed from version 0.3.0
> 13:15:29    Up to date!
> 13:15:29  Installing fivetran/twitter_ads_source
> 13:15:30    Installed from version 0.6.0
> 13:15:30    Up to date!
> 13:15:30  Installing fivetran/amazon_ads_source
> 13:15:31    Installed from version 0.1.1
> 13:15:31    Up to date!
> 13:15:31  Installing fivetran/reddit_ads_source
> 13:15:31    Installed from version 0.1.0
> 13:15:31    Up to date!
> 13:15:31  Installing calogica/dbt_date
> 13:15:32    Installed from version 0.7.2
> 13:15:32    Up to date!
> 13:15:32  
> 13:15:32  Updates available for packages: ['dbt-labs/dbt_project_evaluator', 'dbt-labs/dbt_external_tables', 'fivetran/netsuite', 'fivetran/salesforce', 'fivetran/salesforce_source']                 
> Update your versions in packages.yml, then run dbt deps
> (.venv) user@ConsOle dbt % dbt build --select dbt_artifacts
> 
> 
> 13:16:00  Running with dbt=1.4.5
> 13:16:00  Unable to do partial parsing because saved manifest not found. Starting full parse.
> 13:16:22  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
> There are 1 unused configuration paths:
> - models.facebook_ads_creative_history
> 13:16:22  Found 990 models, 1562 tests, 0 snapshots, 0 analyses, 1286 macros, 1 operation, 42 seed files, 631 sources, 0 exposures, 19 metrics
> 13:16:23  
> 13:16:36  Concurrency: 8 threads (target='dev')
> 13:16:36  
> 13:16:36  1 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.exposures  [RUN]
> 13:16:36  2 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.invocations  [RUN]
> 13:16:36  3 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.model_executions  [RUN]
> 13:16:36  4 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.models .. [RUN]
> 13:16:36  5 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.seed_executions  [RUN]
> 13:16:36  6 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.seeds ... [RUN]
> 13:16:36  7 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.snapshot_executions  [RUN]
> 13:16:36  8 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.snapshots  [RUN]
> 13:16:43  8 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.snapshots  [MERGE (0.0 rows, 0 processed) in 6.55s]
> 13:16:43  9 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.sources . [RUN]
> 13:16:43  3 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.model_executions  [MERGE (0.0 rows, 0 processed) in 6.59s]
> 13:16:43  10 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.test_executions  [RUN]
> 13:16:43  7 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.snapshot_executions  [MERGE (0.0 rows, 0 processed) in 6.75s]
> 13:16:43  11 of 34 START sql incremental model dbt_user_t_dbt_artifacts_source.tests .. [RUN]
> 13:16:43  4 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.models  [MERGE (0.0 rows, 0 processed) in 6.76s]
> 13:16:43  12 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__snapshots  [RUN]
> 13:16:43  5 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.seed_executions  [MERGE (0.0 rows, 0 processed) in 6.80s]
> 13:16:43  13 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__model_executions  [RUN]
> 13:16:43  2 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.invocations  [MERGE (0.0 rows, 0 processed) in 7.02s]
> 13:16:43  14 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__snapshot_executions  [RUN]
> 13:16:43  1 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.exposures  [MERGE (0.0 rows, 0 processed) in 7.13s]
> 13:16:43  15 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__models  [RUN]
> 13:16:44  6 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.seeds  [MERGE (0.0 rows, 0 processed) in 7.29s]
> 13:16:44  16 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__seed_executions  [RUN]
> 13:16:49  9 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.sources  [MERGE (0.0 rows, 0 processed) in 6.45s]
> 13:16:49  17 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__invocations  [RUN]
> 13:16:49  10 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.test_executions  [MERGE (0.0 rows, 0 processed) in 6.65s]
> 13:16:49  18 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__exposures  [RUN]
> 13:16:50  11 of 34 OK created sql incremental model dbt_user_t_dbt_artifacts_source.tests  [MERGE (0.0 rows, 0 processed) in 6.98s]
> 13:16:50  19 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__seeds  [RUN]
> 13:16:51  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:7d252a7f-2440-4653-b5f1-8b2528d22142&page=queryresults
> 13:16:51  12 of 34 ERROR creating sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__snapshots  [ERROR in 7.66s]
> 13:16:51  20 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__sources  [RUN]
> 13:16:51  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:bbb3842d-ae27-4d7c-9f61-3596477fc63e&page=queryresults
> 13:16:51  15 of 34 ERROR creating sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__models  [ERROR in 7.78s]
> 13:16:51  21 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__test_executions  [RUN]
> 13:16:52  17 of 34 OK created sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__invocations  [CREATE VIEW (0 processed) in 2.33s]
> 13:16:52  22 of 34 START sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__tests  [RUN]
> 13:16:53  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:0d15751b-4a82-45e0-8f37-7129d4e83293&page=queryresults
> 13:16:53  13 of 34 ERROR creating sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__model_executions  [ERROR in 10.04s]
> 13:16:53  23 of 34 SKIP relation dbt_user_t_dbt_artifacts.dim_dbt__snapshots .......... [SKIP]
> 13:16:53  24 of 34 SKIP relation dbt_user_t_dbt_artifacts.dim_dbt__models ............. [SKIP]
> 13:16:53  25 of 34 START sql view model dbt_user_t_dbt_artifacts.fct_dbt__invocations . [RUN]
> 13:16:53  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:ccb2b12b-7e47-4a3b-ac8c-c78091693cf9&page=queryresults
> 13:16:53  16 of 34 ERROR creating sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__seed_executions  [ERROR in 9.71s]
> 13:16:53  26 of 34 SKIP relation dbt_user_t_dbt_artifacts.dim_dbt__current_models ..... [SKIP]
> 13:16:53  27 of 34 SKIP relation dbt_user_t_dbt_artifacts.fct_dbt__model_executions ... [SKIP]
> 13:16:53  28 of 34 SKIP relation dbt_user_t_dbt_artifacts.fct_dbt__seed_executions .... [SKIP]
> 13:16:54  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:3aee39eb-5131-4af1-bb77-9e6b000cf28f&page=queryresults
> 13:16:54  14 of 34 ERROR creating sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__snapshot_executions  [ERROR in 10.33s]
> 13:16:54  29 of 34 SKIP relation dbt_user_t_dbt_artifacts.fct_dbt__snapshot_executions  [SKIP]
> 13:16:56  25 of 34 OK created sql view model dbt_user_t_dbt_artifacts.fct_dbt__invocations  [CREATE VIEW (0 processed) in 2.47s]
> 13:16:58  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:e6441a60-86e9-4abe-8f0b-cd28f902981c&page=queryresults
> 13:16:58  20 of 34 ERROR creating sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__sources  [ERROR in 7.28s]
> 13:16:58  30 of 34 SKIP relation dbt_user_t_dbt_artifacts.dim_dbt__sources ............ [SKIP]
> 13:16:59  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:16ecb8b6-38c0-4f23-b22b-850cc928201d&page=queryresults
> 13:16:59  19 of 34 ERROR creating sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__seeds  [ERROR in 8.76s]
> 13:16:59  31 of 34 SKIP relation dbt_user_t_dbt_artifacts.dim_dbt__seeds .............. [SKIP]
> 13:17:01  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:cc9957ba-4f36-482d-989f-5a0d2c9ed48f&page=queryresults
> 13:17:01  21 of 34 ERROR creating sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__test_executions  [ERROR in 9.53s]
> 13:17:01  32 of 34 SKIP relation dbt_user_t_dbt_artifacts.fct_dbt__test_executions .... [SKIP]
> 13:17:02  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:290b641c-05ff-4d1f-976c-c3d66d58aa00&page=queryresults
> 13:17:02  18 of 34 ERROR creating sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__exposures  [ERROR in 12.24s]
> 13:17:02  33 of 34 SKIP relation dbt_user_t_dbt_artifacts.dim_dbt__exposures .......... [SKIP]
> 13:17:03  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:928664f5-1966-4e7f-9870-efa9c3a77861&page=queryresults
> 13:17:03  22 of 34 ERROR creating sql view model dbt_user_t_dbt_artifacts_staging.stg_dbt__tests  [ERROR in 11.16s]
> 13:17:03  34 of 34 SKIP relation dbt_user_t_dbt_artifacts.dim_dbt__tests .............. [SKIP]
> 13:17:03  
> 13:17:03  Running 1 on-run-end hook
> 13:17:03  Uploading model executions
> 13:17:12  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prj-dlake&j=bq:US:03dce70d-4bfc-48b1-8b42-7308e34d430b&page=queryresults
> 13:17:12  Database error while running on-run-end
> 13:17:12  
> 13:17:12  Finished running 11 incremental models, 23 view models in 0 hours 0 minutes and 48.99 seconds (48.99s).
> 13:17:12  
> 13:17:12  Completed with 11 errors and 0 warnings:
> 13:17:12  
> 13:17:12  Database Error in model stg_dbt__snapshots (models/staging/stg_dbt__snapshots.sql)
> 13:17:12    Syntax error: Expected ")" but got "(" at [19:148]
> 13:17:12    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__snapshots.sql
> 13:17:12  
> 13:17:12  Database Error in model stg_dbt__models (models/staging/stg_dbt__models.sql)
> 13:17:12    Syntax error: Expected ")" but got "(" at [19:148]
> 13:17:12    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__models.sql
> 13:17:12  
> 13:17:12  Database Error in model stg_dbt__model_executions (models/staging/stg_dbt__model_executions.sql)
> 13:17:12    Syntax error: Expected ")" but got "(" at [19:148]
> 13:17:12    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__model_executions.sql
> 13:17:12  
> 13:17:12  Database Error in model stg_dbt__seed_executions (models/staging/stg_dbt__seed_executions.sql)
> 13:17:12    Syntax error: Expected ")" but got "(" at [19:148]
> 13:17:12    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__seed_executions.sql
> 13:17:12  
> 13:17:12  Database Error in model stg_dbt__snapshot_executions (models/staging/stg_dbt__snapshot_executions.sql)
> 13:17:12    Syntax error: Expected ")" but got "(" at [19:148]
> 13:17:12    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__snapshot_executions.sql
> 13:17:12  
> 13:17:12  Database Error in model stg_dbt__sources (models/staging/stg_dbt__sources.sql)
> 13:17:12    Syntax error: Expected ")" but got "(" at [19:148]
> 13:17:12    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__sources.sql
> 13:17:12  
> 13:17:12  Database Error in model stg_dbt__seeds (models/staging/stg_dbt__seeds.sql)
> 13:17:12    Syntax error: Expected ")" but got "(" at [19:148]
> 13:17:12    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__seeds.sql
> 13:17:12  
> 13:17:12  Database Error in model stg_dbt__test_executions (models/staging/stg_dbt__test_executions.sql)
> 13:17:12    Syntax error: Expected ")" but got "(" at [19:148]
> 13:17:12    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__test_executions.sql
> 13:17:12  
> 13:17:12  Database Error in model stg_dbt__exposures (models/staging/stg_dbt__exposures.sql)
> 13:17:12    Syntax error: Expected ")" but got "(" at [19:148]
> 13:17:12    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__exposures.sql
> 13:17:12  
> 13:17:12  Database Error in model stg_dbt__tests (models/staging/stg_dbt__tests.sql)
> 13:17:12    Syntax error: Expected ")" but got "(" at [19:148]
> 13:17:12    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__tests.sql
> 13:17:12  
> 13:17:12  on-run-end failed, error:
> 13:17:12   Syntax error: Unclosed string literal at [409:13]
> 13:17:12  
> 13:17:12  Done. PASS=13 WARN=0 ERROR=11 SKIP=11 TOTAL=35

full log is here do you want my requirement.txt as well ?

**Comment by calkan-cue** [comment]
Sadly this issue was because of the name conflict of the hash macro in our custom code and package it seems fixed. Sorry for false alarm
